torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -315,7 +315,7 @@ def torch_dtype_to_ttnn_dtype(dtype: torch.dtype):
 # RuntimeError: TT_FATAL @ ../tt_metal/impl/buffers/buffer.cpp:38: valid_page_size
 # For valid non-interleaved buffers page size 2048 must equal buffer size X. For interleaved-buffers page size should be divisible by buffer size
 def has_valid_page_size(shape):
-    return len(shape) >= 2 and ((shape[-1] > 1 and shape[-1] < 31) or (shape[-1] >= 32 and (shape[-1] % 32 == 0)))
+    return len(shape) >= 2 and shape[-1] > 0 and shape[-1] % 32 == 0
```

Comment:
Innermost size in [1, 31] is no longer considered valid paging in tt-metal
https://github.com/tenstorrent/tt-metal/blob/6fbf494df30b6e228847bcd94574ca7a9a832e0d/tt_metal/impl/buffers/buffer.cpp#L25-L48

Everything should be fine now.  The conversion is pretty restricted though, since it only applies for tensors of valid page sizes.

Reply:
Please add this info to the ticket. Want to make sure we track the issue preventing this from full conversion